src/automations/components/AutomationActionSendNotificationInput.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <p-content class="automation-action-send-notification">
+    <p-label label="Block" :state="blockDocumentState" :message="blockDocumentError">
+      <BlockCapabilityBlockDocumentInput v-model="blockDocumentId" :state="blockDocumentState" capability="notify" required use-modal />
+    </p-label>
+
+    <template v-if="isAnonymousBlock">
+      <p-message info>
+        This notification action is using an anonymous block. This is most likely because it was migrated from an existing notification.
+      </p-message>
+    </template>
+
+    <p-label label="Subject" :state="subjectState" :message="subjectError">
+      <p-code-input v-model="subject" :state="subjectState" lang="jinja" />
+    </p-label>
+
+    <p-label label="Body" :state="bodyState" :message="bodyError">
+      <p-code-input v-model="body" :state="bodyState" rows="5" lang="jinja" />
+    </p-label>
+
+    <p-message info>
+      Event fields and any of the following objects present in the triggering event can be used in notification templates: flow, deployment, flow run, work pool, work queue, metric.
```

Comment:
```suggestion
      In addition to any fields present on the triggering event, the following objects can be used in notification templates: <p-code inline>flow</p-code>, <p-code inline>deployment</p-code>, <p-code inline>flow_run</p-code>, <p-code inline>work_pool</p-code>, <p-code inline>work_queue</p-code>, and <p-code inline>metric</p-code>.
```

Check me on the specifics but I think this language is a little clearer and hopefully the code snippets provide a better jumping-off point.

Reply:
Sounds good to me.